### PR TITLE
Brendan/npm dependencies generator

### PR DESCRIPTION
**Pull request by brendanfalk**
NPM uninstall suggestions
* description: now the version number
* name/displayName: now the full key, rather than stripping out the scope

### CODE REVIEW

**Comment by fedeci** [comment]
#477😆

**Comment by QuiiBz** [comment]
As @fedeci mentioned, looks like #477 contains the same changes, and it has already been merged. Should we close this one or is there another feature that I missed in this PR?

File: dev/npm.ts [hunk 1]
```diff
@@ -25,30 +25,36 @@ const searchGenerator: Fig.Generator = {
 };
 
 const dependenciesGenerator: Fig.Generator = {
-  script: "cat package.json",
+  script:
+    "until [[ -f package.json ]] || [[ $PWD = '/' ]]; do cd ..; done; cat package.json",
```
Comment (fedeci) [inline dev/npm.ts:29]: @QuiiBz this one seems like a good feature to add.

Comment (QuiiBz) [reply dev/npm.ts:29]: Good catch!

Comment (Nutlope) [reply dev/npm.ts:29]: Ah, great catch! Going to close this PR and re-submit one with this single change

**Comment by Nutlope** [comment]
@brendanfalk I'm closing this PR since @fedeci submitted a similar one

**Pull request closed without merging**